dev/com.ibm.ws.app.manager.springboot/resources/com/ibm/ws/app/manager/springboot/internal/resources/Messages.nlsprops

```diff
@@ -36,13 +36,17 @@ warning.could.not.thin.application.explanation=The application could not be conv
 warning.could.not.thin.application.useraction=Attempt to resolve the issue in the exception message and either restart the server or update the application. 
 
 error.need.springboot.version.15=CWWKC0252E: Could not run the application because the wrong version of the Spring Boot feature is configured in server.xml. The application requires the feature springBoot-1.5 to be configured.
-error.need.springboot.version.15.explanation= The application requires Spring Boot version 1.5 but a wrong version of the Spring Boot feature is configured in the server.xml.  
-error.need.springboot.version.15.useraction= The springBoot-1.5 feature needs to be configured in the server.xml.
+error.need.springboot.version.15.explanation=The application requires Spring Boot version 1.5 but a wrong version of the Spring Boot feature is configured in the server.xml.  
+error.need.springboot.version.15.useraction=The springBoot-1.5 feature needs to be configured in the server.xml.
 
 error.need.springboot.version.20=CWWKC0253E: Could not run the application because the wrong version of the Spring Boot feature is configured in server.xml. The application requires the feature springBoot-2.0 to be configured.
-error.need.springboot.version.20.explanation= The application requires Spring Boot version 2.0 but a wrong version of the Spring Boot feature is configured in the server.xml.  
-error.need.springboot.version.20.useraction= The springBoot-2.0 feature needs to be configured in the server.xml.
+error.need.springboot.version.20.explanation=The application requires Spring Boot version 2.0 but a wrong version of the Spring Boot feature is configured in the server.xml.  
+error.need.springboot.version.20.useraction=The springBoot-2.0 feature needs to be configured in the server.xml.
 
```

Comment:
+error.need.springboot.version.20.explanation=The application requires Spring Boot version **2.0,** but a wrong version of the Spring Boot feature is configured in the server.xml.